lab_1_classify_by_unigrams/main.py

```diff
@@ -11,6 +11,12 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if not isinstance(text, str):
+        return None
+
+    text = ''.join(element for element in text if element.isalpha())
+    letters = list(text.lower())
```

Comment:
Consider making a list right away instead of firstly making a string and only then converting it